src/citrine/resources/ara_definition.py

```diff
@@ -221,10 +225,10 @@ def get_with_version(self, definition_uid: Union[UUID, str],
 
     def build(self, data: dict) -> AraDefinition:
         """[ALPHA] Build an individual Ara Definition from a dictionary."""
-        defn = AraDefinition.build(data['ara_definition'])
-        defn.definition_uid = data['definition_id']
-        defn.version_number = data['version_number']
-        defn.version_uid = data['id']
+        defn = AraDefinition.build(data['version']['ara_definition'])
```

Comment:
Would it make sense for `version` to be a constant, given the number of times it appears?  I was originally thinking just use `_individual_key`, but that got cleared in this PR.

Reply:
I'll just extract those parts of the dict - it's only in this scope, and used 3 times.